src/index.ts

```diff
@@ -107,9 +107,20 @@ export interface Record {
     valueString(): string
 }
 
+/**
+ * An item that may be sent via the Producer.
+ */
+export type Item = string | ArrayBuffer
```

Comment:
Does this need to be exported? Side note, the name `Item` is just too generic.

Reply:
Hmm it probably does not need to be exported. What might a better name be? Everything sounds either too vague or too specific, `ProducerData`? `ProducerItem`? `KeyValueData`? `ValueData`? `StringOrBuffer`? I don't really know.

The main reason I made it its own type was primarily for documentation purposes

Reply:
I think `ProducerItem` is okay. I still don't love it but don't have anything better.